_documentation/subscriptions/subscription-groups.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Subscription groups / tags / categories
+layout: doc
+level: "2"
+group: "subscriptions"
+icon: "fas fa-tags"
+---
+
+# Assigning groups / tags / categories to subscription
+
+To manage subscriptions it is possible to assign groups / tags / categories to a subscription.
+Please note that for the moment it is not possible to create queue(s) based on it. To do so it would first be needed to support multiple queues. (See related [issue 2648](https://github.com/AntennaPod/AntennaPod/issues/2648))
+
+To assign groups / tags / categoriers to a podcasdt you have to :
```

Comment:
I would try to avoid the word "podcast". While it usually refers to the subscription, more and more users use the same word to refer to episodes. When only using subscription/episode, it is more clear what elements can be tagged.

```suggestion
To assign groups / tags / categories to a subscription you have to :
```

Reply:
I disagree. A podcast is a podcast, an episode an episode. This is notably different from the situation with 'blog' and 'blog (post)'. If institutions aren't using the right terminology they will only help the wider public (who, let's face it, are only starting to get introduced to the concept) to not get familiar with the proper terms.
And the confusion is minor, without the risk of causing severe consequences (e.g. data loss).